main.py

```diff
@@ -1,157 +1,215 @@
-import pygame, sys, random, math
-from pygame.locals import *
+######################################
+# - Fix Basic syntax
+# - Port to Python3
+# - Add Game Restart
+######################################
+import pygame
+import sys
+import random
+import math
+
+from pygame.locals import QUIT, KEYUP
 
 pygame.init()
 
 #setup the window display
-size=(1024, 768)
-windowSurface = pygame.display.set_mode((size), 0, 32)
+size = (1024, 768)
+windowSurface  = pygame.display.set_mode(size, 0, 32)
 pygame.display.set_caption('Super Mumbo Epicness')
 
 # set up fonts
-basicFont = pygame.font.SysFont(None, 48) #none is for default system font, number is size of font
+basicFont = pygame.font.SysFont(None, 48) #None is for default system font
 
 #set colors R, G, B code
 BLACK = (0, 0, 0)
 WHITE = (255, 255, 255)
 
 #images
-cart_img = pygame.image.load('Images/bb.jpg')
-coin_img = pygame.image.load('Images/coin.jpg')
-bluecoin = pygame.image.load('Images/bluecoin.jpg')
-bomb = pygame.image.load('Images/bomb.png')
+#convert for easy blitting
+cart_img = pygame.image.load('Images/bb.jpg').convert()
+coin_img = pygame.image.load('Images/coin.jpg').convert()
+bluecoin = pygame.image.load('Images/bluecoin.jpg').convert()
+bomb = pygame.image.load('Images/bomb.png').convert()
 R = random.randint(1, 4)
-BG = pygame.image.load('Images/coinfallbg'+str(R)+'.jpg')
+BG = pygame.image.load('Images/coinfallbg'+str(R)+'.jpg').convert()
+
+cart_img.fill((255, 0, 0))
+coin_img.fill((255, 255, 0))
+bluecoin.fill((0, 0, 255))
+bomb.fill((125, 125, 125))

```

Comment:
Running the game with the changes shows these colours instead of the images, please check